### PR TITLE
[Test] Unpeer

### PR DESCRIPTION
**Pull request by aleoli**
# Description

This pr adds some checks after the liqo clusters unpeer

### CODE REVIEW

File: test/e2e/pipeline/installer/liqoctl/unpeer.sh [hunk 1]
```diff
@@ -36,3 +48,44 @@ do
...
+
+  check_no_resources "${KUBECTL} get tenants.authentication.liqo.io" "Tenants"
+  check_no_resources "${KUBECTL} get identities.authentication.liqo.io -A" "Identities"
+  check_no_resources "${KUBECTL} get resourceslices.authentication.liqo.io -A" "ResourceSlices"
+
+  check_no_resources "${KUBECTL} get gatewayclients.networking.liqo.io -A" "GatewayClients"
+  check_no_resources "${KUBECTL} get gatewayservers.networking.liqo.io -A" "GatewayServers"
+  check_no_resources "${KUBECTL} get publickeies.networking.liqo.io -A" "PublicKeys"
+  check_no_resources "${KUBECTL} get wggatewayclients.networking.liqo.io -A" "WgGatewayClients"
+  check_no_resources "${KUBECTL} get wggatewayservers.networking.liqo.io -A" "WgGatewayServers"
```
Comment (fra98) [inline test/e2e/pipeline/installer/liqoctl/unpeer.sh:65]: I would add connections and configurations

**Review by fra98 — APPROVED** [review]

**Review by cheina97 — APPROVED** [review]

**Pull request merged**